docs/guidelines/accessibility-label.md

```diff
@@ -149,10 +146,10 @@ Is it possible to specify a list of allowed all caps accessibility labels, [more
 
 ## Related AMA components
 
-- [ExpandablePressable](../components/expandablepressable)
-- [Pressable](../components/pressable)
-- [TouchableOpacity](../components/touchableopacity)
-- [TouchableWithoutFeedback](../components/TouchableWithoutFeedback
+- [ExpandablePressable](/core/components/expandablepressable)
+- [Pressable](/core/components/pressable)
+- [TouchableOpacity](/core/components/touchableopacity)
+- [TouchableWithoutFeedback](/core/components/TouchableWithoutFeedback)
```

Comment:
These are in `react-native` not `core`. Well Pressable is in core but I think it should maybe more to `react-native`